src/components/VDialog/VDialog.js

```diff
@@ -193,14 +193,19 @@ export default {
       }, [this.$slots.activator]))
     }
 
-    const dialog = h('transition', {
-      props: {
-        name: this.transition || '', // If false, show nothing
-        origin: this.origin
-      }
-    }, [h('div', data,
-      this.showLazyContent(this.$slots.default)
-    )])
+    var dialog
+    if (this.transition) {
+      dialog = h('transition', {
```

Comment:
The only thing that really changes here is the tag and props, let's see if we can simplify this a bit.

Reply:
Done!

I looked at unit tests, but since the change fixes a UI latency issue it's difficult to functional test.